lib/container/container.go

```diff
@@ -133,6 +133,25 @@ func WaitContainer(id string) (int, error) {
 	return client.WaitContainer(id)
 }
 
+func getNewDockerClient() (*docker.Client, error) {
+	if os.Getenv("DOCKER_TLS_VERIFY") == "1" {
```

Comment:
the only requirement is that DOCKER_TLS_VERIFY is set, ie `os.Getenv("DOCKER_TLS_VERIFY") != ""`